test/gui/tst_syncing/test.feature

```diff
@@ -353,4 +353,13 @@ Feature: Syncing files
         Then as "Alice" folder "folder1" should exist on the server
         And as user "Alice" folder "folder1" should contain "500" items on the server
         And as "Alice" folder "folder2" should exist on the server
-        And as user "Alice" folder "folder2" should contain "500" items on the server
\ No newline at end of file
+        And as user "Alice" folder "folder2" should contain "500" items on the server
+
+
+    Scenario: Choosing local download directory in advanced configuration
+        Given the user has started the client
+        When the user adds the following account information:
+            | server   | %local_server% |
+            | user     | Alice          |
+            | password | 1234           |
+        Then local folder selection option should be visible
```

Comment:
choosing local directory step should be in When step

Reply:
The step is not to choose the folder but assert that the user can see the option to select the sync folder

Reply:
I think scenario title says `Choosing local download directory in advanced configuration` so i got confused